analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentService.java

```diff
@@ -9,15 +9,15 @@
     Response align(Request request);
 
     class Request {
-        private final List<? extends ProjectVersionRef> dependencies;
+        private final Collection<? extends ProjectVersionRef> dependencies;
```

Comment:
Might as well specify Set semantics?

Reply:
Not sure... We might have to change this again in the future so I would rather keep it generic for now, what do you think?